website/docs/blueprints/ai-ml/bionemo.md

```diff
@@ -0,0 +1,225 @@
+3) [**FSx for Lustre CSI Driver**](https://docs.aws.amazon.com/eks/latest/userguide/fsx-csi.html)
+
+
+## Deploying the Example
+
+In this example we are going to deploy an Amazon EKS cluster and run a data preparation job and a distributed model training job.
+
+<CollapsibleContent header={<h3><span>Pre-requisites</span></h3>}>
+
+Ensure that you have installed the following tools on your machine.
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+4. [python3](https://www.python.org/)
+
+</CollapsibleContent>
+
+<CollapsibleContent header={<h3><span>Deploy the EKS Cluster with KubeRay Operator</span></h3>}>
+
+#### Clone the repository
+
+```bash
+git clone https://github.com/awslabs/data-on-eks.git
+```
+
+#### Initialize Terraform
+
+Navigate into the example directory
+
+```bash
+cd data-on-eks/ai-ml/bionemo
+```
+
+#### Run the install script
+
+Use the provided helper script `install.sh` to run the terraform init and apply commands. By default the script deploys EKS cluster to `us-west-2` region. Update `variables.tf` to change the region. This is also the time to update any other input variables or make any other changes to the terraform template.
+
+
+```bash
+./install .sh
+```
+
+</CollapsibleContent>
+
+<CollapsibleContent header={<h3><span>Verify Deployment</span></h3>}>
+
+Update local kubeconfig so we can access kubernetes cluster
+
+```bash
+aws eks update-kubeconfig --name bionemo-cluster #or whatever you used for EKS cluster name
+```
+
+Install Kubeflow Training Operator:
+```bash
+kubectl apply -k "github.com/kubeflow/training-operator/manifests/overlays/standalone?ref=v1.7.0"
+```
+
+First, lets verify that we have worker nodes running in the cluster.
+
+```bash
+kubectl get nodes
+```
+:::info
+```bash
+NAME                                          STATUS   ROLES    AGE   VERSION
+ip-100-64-243-22.us-west-2.compute.internal   Ready    <none>   47h   v1.29.0-eks-5e0fdde
+ip-100-64-70-74.us-west-2.compute.internal    Ready    <none>   47h   v1.29.0-eks-5e0fdde
+```
+:::
+
+Next, lets verify all the pods are running.
+
+```bash
+kubectl get pods -A
+```
+Make sure training-operator, nvidia-device-plugin and fsx-csi-controller pods are running and healthy.
+
+#### Run bionemo jobs
+
+Once we are sure everything is working, we can start submitting jobs to our clusters.
+
+First job we are going to run is uniref50-job which will download and split the data for efficient processing.
+
+```bash
+cd training
+kubectl apply -f uniref50-job.yaml
+```
+
+If you look at the yaml manifest you would see it is a kubernetes Job, which simply runs a python script within the bionemo container.
+
+This process takes relatively a long time (~50-60 hours). In order to see if it is working check the logs from this pod:
```

Comment:
Add some details on why it takes longer. Is this a trianing job? How much data its processing? Is this job processing data from S3 or Fsx etc..